src/components/EmptyArea.tsx

```diff
@@ -0,0 +1,45 @@
+import { Box, Center, VStack, Text, HTMLChakraProps } from "@chakra-ui/react"
```

Comment:
there's nothing wrong with putting this in `src/components` but this implies that the # of use-cases this component has to cover would be much wider than if it was limited. 

this impacts how the component should be structured - eg: presence of button/text. one way people get around this (see: chakra's [menu](https://chakra-ui.com/docs/components/menu#menubutton-props) or [modal](https://chakra-ui.com/docs/components/modal#block-scrolling-when-modal-opens)) is by having many related components grouped together and exported under a common prefix (or as per design system, under a grouped export).

the rationale to allow for more flexibility here is good (by adopting optional arguments) - but this makes a judgement that the text **has to exist**, which might not be ideal. could i check here if this has been cleared by our designers and also, that empty states going forward have been tied down to this

Reply:
Sure! I think this is just something to be aware of next time. Wrt this component, it's general enough to be placed here and the given API is fine (if text is optional).